README.md

```diff
@@ -36,4 +36,12 @@ export default ExampleComponent;
 
 ## Which lifecycles are supported?
 
-Currently, this polyfill supports [static `getDerivedStateFromProps`](https://deploy-preview-587--reactjs.netlify.com/docs/react-component.html#static-getderivedstatefromprops) and [`getSnapshotBeforeUpdate`](https://deploy-preview-587--reactjs.netlify.com/docs/react-component.html#getsnapshotbeforeupdate)- both introduced in version 16.3.
\ No newline at end of file
+Currently, this polyfill supports [static `getDerivedStateFromProps`](https://deploy-preview-587--reactjs.netlify.com/docs/react-component.html#static-getderivedstatefromprops) and [`getSnapshotBeforeUpdate`](https://deploy-preview-587--reactjs.netlify.com/docs/react-component.html#getsnapshotbeforeupdate)- both introduced in version 16.3.
+
+## Validation
+
+Note that in order for the polyfill to work, none of the following lifecycles can be defined by your component: `componentWillMount`, `componentWillReceiveProps`, or `componentWillUpdate`.
+
+Note also that if your component contains `getSnapshotBeforeUpdate`, `componentWillUpdate` must be defined as well.
```

Comment:
Did you mean `componentDidUpdate`?

Reply:
In other words I'd expect the person trying to use `getSnapshotBeforeUpdate` to want to migrate *away* from `componentWillUpdate`, not use it.

Reply:
Yes 😦 This was a typo.